tortoise/ninja_tortoise_test.go

```diff
@@ -896,3 +896,46 @@ func AddLayer(m *mesh.MeshDB, layer *types.Layer) error {
 	}
 	return nil
 }
+
+func TestNinjaTortoise_Recovery(t *testing.T) {
```

Comment:
I think perhaps it's good to have a test that loads data from persistent storage to tortoise until layer x and continues with sync until layer y, WDYT?

Reply:
i agree thats why we have the late nodes test